crates/indexer/src/models/v2_objects.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::token_models::v2_token_utils::V2TokenResource;
+use crate::{
+    models::move_resources::MoveResource,
+    schema::{current_objects, objects},
+    util::standardize_address,
+};
+use aptos_api_types::{DeleteResource, Transaction, WriteResource, WriteSetChange};
+use bigdecimal::BigDecimal;
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Deserialize, FieldCount, Identifiable, Insertable, Serialize)]
+#[diesel(primary_key(transaction_version, write_set_change_index))]
+#[diesel(table_name = objects)]
+pub struct Object {
```

Comment:
I thought we also want to track the object's "types" - i.e. all the resources defined in the ObjectCore resource group?

Reply:
Maybe, but I prolly need to do it somewhere else not here.